### PR TITLE
Resolve GitHub issue #64: "Maintain a list of frontend grid sizes and export them"

### PR DESCRIPTION
**Pull request by metaa**
This is an attempt to resolve issue #64 "Maintain a list of frontend grid sizes and export them".
I'm not sure if the method of how this is implemented is the best way, but I think it's okay.
I made it so that the information exported cannot be overridden.

### CODE REVIEW

File: src/InteractiveConstants.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+            height: 40,
+        },
+    ];
+
+	/**
+     * @returns {{GridLayout[]}} a copy of the stored layout definitions to prevent users from breaking it.
+	 */
+	public static get gridLayoutSizes(): IGridLayout[] {
+        return {...InteractiveConstants._gridLayoutSizes};
+    }
```
Comment (connor4312) [inline src/InteractiveConstants.ts]: You can use Object.freeze for the same effect, which returns a `ReadOnlyArray` for compile-time enforcement for immutability.

Comment (metaa) [inline src/InteractiveConstants.ts]: Ohhh, right, why didn't I think of that ...
I'll improve that once I'm home 😬

File: src/InteractiveConstants.ts [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import { IGridLayout } from './state/interfaces/controls';
+
+/**
+ * Offers constant information values to use in an application.
+ */
+export class InteractiveConstants {
+    private static readonly _gridLayoutSizes: IGridLayout[] = [
```
Comment (ProbablePrime) [inline src/InteractiveConstants.ts]: Does this need to be a class?

Could it be a module instead?

Comment (ProbablePrime) [inline src/InteractiveConstants.ts]: Checkout `InteractiveError` in errors.ts for an example of a module.

File: src/constants.ts [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import { IGridLayout } from './state/interfaces/controls';
+
+/**
+ * Offers constant information values to use in an application.
+ */
+export module InteractiveConstants {
```
Comment (SimonSchick) [inline src/constants.ts]: What is the reason for wrapping this inside of a module?

Comment (metaa) [inline src/constants.ts]: Where else could you place such information? 🤔 As a static method in Scenes?
Also that way other constants for stuff could be grouped.

Comment (SimonSchick) [inline src/constants.ts]: The file is already called `constants` why not just export it directly?

Comment (metaa) [inline src/constants.ts]: I figured that if there's more constants stuff to be exported in the future, you'd wanna wrap it anyways 😊
Makes it a bit cleaner in my opinion.

Comment (SimonSchick) [inline src/constants.ts]: We don't really use modules to declare to `wrap things up` anywhere in TS, namespaces maybe, but not modules.

Comment (metaa) [inline src/constants.ts]: Alright, feel free to change it then!
I won't be able to for the next two weeks.

Comment (SimonSchick) [inline src/constants.ts]: I will merge and change it after.

**Pull request merged**